src/tools/bidirectionalTool/utils/getLongestAndShortestDiameters.js

```diff
@@ -1,16 +0,0 @@
-export default function (handles, image = {}) {
```

Comment:
I think we should move all these `BidirectionalTool` helpers to `src/util/bidirectional`.

Reply:
Or put all those other tool-specific utils sit here in `src/tools` also? As long as we are consistent so one doesn't have to go hunting around for implementation.

Reply:
@JamesAPetts do you have a strong preference either way? I'm more than happy to do the change.

Reply:
Maybe a compromise: Put them all in `src/tools/utils/toolSpecificFolder`? Just to reduce the number of top level folders in `src/tools`. They are more "implementations" than "utils", they aren't re-used, just separated when a class gets too big.